util.go

```diff
@@ -46,3 +47,26 @@ func FulfillmentPathPrefix(resource string, resourceID int64) string {
 	}
 	return prefix
 }
+
+type OnlyDate struct {
```

Comment:
Thanks for putting this in util !
Sure we don't have anything like this in any of the other resource services?

Reply:
And can you add a few basic tests for this? 
I don't see the date field getting properly covered in the tests but it would also make sense to have just a few basic happy path / error path tests just for the type in util_test.go

Reply:
@oliver006 we only have something remotely similar here: https://github.com/bold-commerce/go-shopify/blob/master/recurringapplicationcharge.go

The difference is those fields are sometimes RFC3339, and sometimes just date (based on the comments there) so it might be different from what we are doing here with `OnlyDate`. Also, let me know if you have a better name, I wanted to make it very clear what the struct is 😄 

I'll add some tests today.